common/src/main/java/com/sequenceiq/cloudbreak/util/FreeIpaPasswordUtil.java

```diff
@@ -24,8 +26,9 @@ private FreeIpaPasswordUtil() {
     public static String generatePassword() {
         String upperCaseLetters = RandomStringUtils.random(PWD_PART_LENGTH, UPPER_CASE_LETTERS_START, UPPER_CASE_LETTERS_END, true, true);
         String lowerCaseLetters = RandomStringUtils.random(PWD_PART_LENGTH, LOWER_CASE_LETTERS_START, LOWER_CASE_LETTERS_END, true, true);
+        String pwdPrefix = RandomStringUtils.random(PWD_PREFIX_LENGTH, LOWER_CASE_LETTERS_START, LOWER_CASE_LETTERS_END, true, true);
         String numbers = RandomStringUtils.randomNumeric(PWD_PART_LENGTH);
         String raw = upperCaseLetters.concat(lowerCaseLetters).concat(numbers).concat(SPECIAL_CHARS);
-        return RandomStringUtils.random(PASSWORD_LENGTH, raw);
+        return pwdPrefix.concat(RandomStringUtils.random(PASSWORD_LENGTH, raw));
```

Comment:
this will produce a password with 35 character length, while we have a `PASSWORD_LENGTH = 32` declaration, which can be misleading